### ISSUE TITLE
First Install File Permission Errors

### ISSUE THREAD

**Issue by Akkarinage** [rathena/FluxCP#61]
Quite often we see topics on the forums about "no permissions to data/tmp/*.php".

It'd be nice if we can output a reasonable suggestion rather than the almighty fluxcp error page. I know it's kinda self-explanatory, but people can't read.

**Comment by RagnarokNova** [comment]
Agreed. It'd be nice if it actually told you what to do instead of having to google things a novice won't understand. I know I struggled a while when I first started to figure out what the error messages were trying to tell me.

**Comment by KeiroD** [comment]
I'd noticed that the error page started listing the suggestions. For me, it was mostly that I needed to temporarily set `777` on `data`. Then I realized it was still having issues and noticed that the error page was displaying `www-data`. Naturally as soon as I saw that, I properly fixed the issue by `chown www-data:www-data data/ -Rv`.

Looks okay though, to me since your error page did properly display how to correct the issue. (Also, apologies for spamming comments on various bits and stuff here. You'd asked for input on all of this on the Forums.)
